Java/src/main/java/epam/Planes/Plane.java

```diff
@@ -0,0 +1,55 @@
+package epam.Planes;
+
+import java.util.Objects;
+
+
+abstract public class Plane {
+	String model;
+	private int maxSpeed;
+	private int maxFlightDistance;
+	private int maxLoadCapacity;
+
+	public Plane(String model, int maxSpeed, int maxFlightDistance, int maxLoadCapacity) {
```

Comment:
add null checks for the string parameters and Zero check for int parameters, in theory the model cannot be null and maxSpeed, maxFlighDistance and maxLoadCapatity cannot not be Zero, this can lead to have side effects if the parameters are not validated, throw exceptions if the parameters are not valid.

Reply:
Done